### PR TITLE
TELCODOCS-1470: DS and RN: TELCODOCS-1458 Expanding support for SNO on cloud providers

### PR DESCRIPTION
**Pull request by johnwilkins**
Additional supported cloud providers for SNO.

Fixes: [TELCODOCS-1470](https://issues.redhat.com//browse/TELCODOCS-1470)

See https://issues.redhat.com/browse/TELCODOCS-1470 for additional details.

Preview URL: http://184.23.213.161:8080/TELCODOCS-1470/installing/installing_sno/install-sno-installing-sno.html#installing-single-node-openshift-on-cloud-providers
For release(s): 4.14, 4.15
QE Review: 

- [x] QE has approved this change. 

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

**Comment by johnwilkins** [comment]
@omertuc Can you have a look at this and provide an LGTM if it meets the requirements?

File: modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-sno-on-a-cloud-provider_{context}"]
+= Additional requirements for installing {sno} on a cloud provider
+
+The documentation for installer-provisioned installation on cloud providers is written with a high availability cluster consisting of three control plane nodes. When referring to the documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
```
Comment (omertuc) [inline modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc:9]: >The documentation for installer-provisioned installation on cloud providers is written with a high availability cluster consisting of three control plane nodes

This sentence feels incomplete. Should "in mind" follow? Or maybe "with" should be replaced with "for"?

**Comment by omertuc** [comment]
>@omertuc Can you have a look at this and provide an LGTM if it meets the requirements?

From what I understand, it seems to me this PR is just moving things around to make the text / layout more generic with "AWS" vs "Cloud providers". 

Is there any new substantial text that I might have missed that needs a technical review?

**Comment by johnwilkins** [comment]
> > @omertuc Can you have a look at this and provide an LGTM if it meets the requirements?
> 
> From what I understand, it seems to me this PR is just moving things around to make the text / layout more generic with "AWS" vs "Cloud providers".
> 
> Is there any new substantial text that I might have missed that needs a technical review?

If there are any additional specific steps for those cloud providers, it wasn't included in the JIRA ticket. So you are correct that this is making the original AWS procedure generic, adds a separate module indicating which providers and CPU architectures are supported, and provides links to the already tested procedures for Azure and GCP.

File: modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc (hunk 1 above)
Comment (mburke5678) [inline modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc:9]: Not sure why I don't like _is written with_ here.  It just sounds odd (like, this comment it written with a laptop).
```suggestion
The documentation for installer-provisioned installation on cloud providers is based on a high availability cluster consisting of three control plane nodes. When referring to the documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
```

Comment (omertuc) [reply modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc:9]: Note that this is copied from a different section of the docs, and is not new. It bothered me as well but I feel like this is irrelevant to this PR

File: modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+* The required machines for cluster installation in cloud provider documentation indicates a temporary bootstrap machine, three control plane machines, and at least two compute machines. You require only a temporary bootstrap machine and one cloud instance for the control plane node and no worker nodes.
```
Comment (mburke5678) [inline modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc:11]:
```suggestion
* For a high availability cluster, the documentation requires a temporary bootstrap machine, three control plane machines, and at least two compute machines. For a {sno} cluster, you need only a temporary bootstrap machine and one cloud instance for the control plane node and no worker nodes.
```

File: installing/installing_sno/install-sno-installing-sno.adoc [hunk 3]
```diff
@@ -59,9 +59,11 @@ include::modules/install-sno-monitoring-the-installation-manually.adoc[leveloffs
 
 * xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to {sno} clusters]
 
-== Installing {sno} on AWS
+== Installing {sno} on cloud providers
```
Comment (mburke5678) [inline installing/installing_sno/install-sno-installing-sno.adoc:62]: I see this was an existing heading; but I think you need an ID string here.

File: modules/install-sno-supported-cloud-providers-for-single-node-openshift.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+The following table contains a list of supported cloud providers and CPU architectures.
+
+.Supported cloud providers
+[options="header"]
+|====
+|Cloud provider |CPU architecture
+|Amazon Web Service (AWS)|x86-64 and ARM
+|Microsoft Azure|x86-64
+|Google Cloud Platform (GCP) | x86-64 and ARM
```
Comment (kelbrown20) [inline modules/install-sno-supported-cloud-providers-for-single-node-openshift.adoc:17]: In this case, and for AWS, I think you can also use [this format if you want](https://redhat-documentation.github.io/supplementary-style-guide/#_numbers)
```suggestion
|Google Cloud Platform (GCP) | 64-bit x86 and 64-bit ARM
```
up to you! whichever you prefer.

**Review by kelbrown20 — COMMENTED** [review]
Hi John! Just a format nit before merging, great work!

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**